src/main/java/vg/civcraft/mc/civchat2/commands/WhoAmI.java

```diff
@@ -0,0 +1,36 @@
+package vg.civcraft.mc.civchat2.commands;
+
+import com.google.common.base.Strings;
+import java.util.ArrayList;
+import java.util.List;
+import net.md_5.bungee.api.ChatColor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import vg.civcraft.mc.civmodcore.command.CivCommand;
+import vg.civcraft.mc.civmodcore.command.StandaloneCommand;
+import vg.civcraft.mc.namelayer.NameAPI;
+
+@CivCommand(id = "whoami")
+public class WhoAmI extends StandaloneCommand {
+
+	@Override
+	public boolean execute(CommandSender sender, String[] arguments) {
+		final String response = ChatColor.YELLOW + "You are: " + ChatColor.RESET;
+		if (sender instanceof Player) {
```

Comment:
Kinda weird to explicitly allow this command for server console in the plugin.yml, but then make it not work for it, it'd just say `You are: `. I'd just disallow it for non-players?

Reply:
What do you mean? Does the console not have a name?

Reply:
Ah nevermind I cant read